### PR TITLE
[IMP] project: open expanded view of project/task in Gantt/Calendar 

### PR DESCRIPTION
**Pull request by rajp-odoo**
Currently, when user open a task or project in the calendar or Gantt view, they cannot access the chatter. With this commit, they can now access the form view through the expanded button.

Task: 3837229

### CODE REVIEW

**Comment by ppr-odoo** [comment]
thanks for the work @rajp-odoo  

The expanded feature is intended for the Gantt view, which is in the Enterprise. Therefore, this code will not be useful for the community. We need to move this code to the Enterprise.

File: addons/project/views/project_project_views.xml [hunk 1]
```diff
@@ -558,13 +558,33 @@
...
+            <field name="name">project.project.view.form.calendar</field>
+            <field name="model">project.project</field>
+            <field name="inherit_id" ref="edit_project"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//sheet" position="after">
+                    <footer>
+                        <button string="Save &amp; Close" special="save" data-hotkey="q" class="btn btn-primary" close="1"/>
+                        <button string="Discard" special="cancel" data-hotkey="x" class="btn btn-secondary" close="1"/>
+                        <button name="action_view_project" type="object" title="View task"
```
Comment (damrOdoo) [inline addons/project/views/project_project_views.xml:571]: I missed it on first review, but shouldn't this button title be 'view project' instead ?

File: addons/project/static/tests/project_models.js [hunk 2]
```diff
@@ -4,10 +4,13 @@ export class ProjectProject extends models.Model {
     _name = "project.project";
 
     name = fields.Char();
+    planned_date_begin = fields.Datetime({ string: "Start Date" });
+    planned_date_stop = fields.Datetime({ string: "stop Date" });
+
```
Comment (damrOdoo) [inline addons/project/static/tests/project_models.js:9]: nitpick, but the extra blank line is not needed imo.

**Review by damrOdoo — COMMENTED** [review]
Thanks for your work. I left two small comment then its ready for the next stage.

**Review by damrOdoo — COMMENTED** [review]
lgtm.

**Comment by xavierbol** [comment]
Will be handled by the framework.

**Pull request closed without merging**